### ISSUE TITLE
CoP Promyvion-Dem bcnm fight not allowing group to enter when group has members who have already cleared.

### ISSUE THREAD

**Issue by Aslipperyotter** [EdenServer/community#3593]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)

### Details

-Had group of six. Two members had already cleared the fight, the other four had not.
-When trying to enter the bcnm, the option to enter was not given.
-Disbanded group, those who were on fight all had the option to enter.
-Reforming the group resulted again in not being eligible to enter. Changing who the leader was had no effect.
-Ended up having the two who cleared the fight only accept party invite after the menu to enter was open.

**Comment by Friesaredone** [comment]
https://github.com/EdenServer/community/issues/911

This was a thing years back when I did my clears as well. Never could get a straight answer from anyone as to whether this was meant to be 'Era' or 'Eden' specific. Appears the ticket was closed w/o so much as a dev comment. 

The only solution was to clear all three base Promyvions before you got the option to go back and help others.

**Comment by 59blargedy** [comment]
you have been able to help after clearing only one or two spires in the past. this is a newer issue that prevents anyone in between certain specific missions from being able to assist that i believe is related to the change in our bcnm systems a while back. it is not related to the previous ticket which was resolved when the solution was merged. 

we've removed that automated link and now manually close tickets with a comment when we fix them, in part due to assumptions like the above.  either way, it's properly reported here so that it can be resolved.
